backend/src/main/java/de/neuefische/backend/controller/TravellerController.java

```diff
@@ -0,0 +1,37 @@
+package de.neuefische.backend.controller;
+
+import de.neuefische.backend.model.Traveller;
+import de.neuefische.backend.model.TravellerRequest;
+import de.neuefische.backend.service.TravellerService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/traveller")
```

Comment:
```suggestion
@RequestMapping("/api/travellers")
```
I'd suggest using plural nouns to be more RESTful.
It's definetely debatable, but I tend to see [some favor](https://stackoverflow.com/questions/6845772/should-i-use-singular-or-plural-name-convention-for-rest-resources) towards the plural convention and find it more logical.


(And yes, I noticed that @danielschwarz-neuefische seems to be in "team-single-nouns" ;D - Let's chat about it next week? :coffee: )